### DISCUSSION TITLE
Erasing bug

### DISCUSSION THREAD

**Discussion by otaworks** [keiwando/toonsquid#425] [General]
I came across this bug while trying to quickly erase an area by selecting and cutting.

I’m drawing on the top layer, and getting towards the end, I turn the top layer on and off, and the hairline goes off when the layer is turned off, but when I try to erase, it doesn’t work.

![RPReplay_Final1696202494(1)](https://github.com/keiwando/toonsquid/assets/114658036/aead8c55-778c-4e22-9c23-1338162cca41)

**Comment by ota-works** [comment]
https://github.com/keiwando/toonsquid/assets/34046385/a29f4698-6f4b-4fab-a274-ac083c61d31a

**Reply by keiwando** [reply]
Could it be that the drawing on frame 16-17 contains a non-pixel layer, which gets auto-selected when you switch to that frame instead of the pixel layer with the lines that you think you are erasing on?

**Comment by ota-works** [comment]
Sorry. I seemed to have used a different account on my iPad to submit the first message
